app/controllers/sitemap_controller.rb

```diff
@@ -16,12 +16,14 @@ class SitemapController < ActionController::Metal
   CACHE_HEADER = "X-Sitemap-Cache"
 
   def sitemap
-    com = community(request)
-    reason = redirect_reason(request)
+    return if render_sitemap_disabled
+
+    com = community_from_request(request)
+    reason = redirect_reason_from_request(request)
 
-    return if do_host_redirect!(com, reason)
     return if render_not_found!(reason)
-    return unless can_show_sitemap?(com)
+    return if render_private_community(com)
```

Comment:
`render_not_found!` has `!`, while the other `render_*` methods don't. I think it's better if all have the exclamation mark, as the behaviour is similar.